operator/pkg/name/name.go

```diff
@@ -49,8 +49,15 @@ const (
 	IngressComponentName ComponentName = "IngressGateways"
 	EgressComponentName  ComponentName = "EgressGateways"
 
-	// Addon components
-	AddonComponentName ComponentName = "Addon"
+	// Addon root component
+	AddonComponentName ComponentName = "AddonComponents"
+
+	// Legacy addon components
+	PrometheusComponentName ComponentName = "Prometheus"
+	KialiComponentName      ComponentName = "Kiali"
+	GrafanaComponentName    ComponentName = "Grafana"
+	TracingComponentName    ComponentName = "Tracing"
+	CoreDNSComponentName    ComponentName = "Istiocoredns"
```

Comment:
still seems to be here. anyway, since the patch is in master you can cherry pick that one separately afterwards if you prefer, this is just a heads up. lmk and i will approve.

Reply:
@ostromart this is naming listed as const here, I tagged them as "legacy" name though, but the actual code to check enablement, those stuff were moved to translate package already.

I can do a further cleanup later.